Examples/Tests/photon_photon_collisions/analysis_base.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+
+import re
+import sys
+
+import numpy as np
+import yt
+from scipy.constants import c, m_e
+
+# default relative tolerance
+rtol = 5e-10
```

Comment:
After rebasing and fixing build errors and runtime errors, this needed to be changed from `5e-11` to `5e-10`  for the tests to pass.